src/Microsoft.AspNetCore.Razor.Evolution/TagHelpers/TagHelperFeature.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Razor.Evolution.Legacy;
+
+namespace Microsoft.AspNetCore.Razor.Evolution.TagHelpers
+{
+    internal class TagHelperFeature : IRazorEngineFeature
```

Comment:
Interesting, you envisioning flowing `ITHDR` in a different manner? Only reason why I felt that it needed to be public was its coupling to `ITHDR`.